src/utils/types.ts

```diff
@@ -282,6 +292,7 @@ interface PageNEW {
 type PageElement = {
   element: ElementBaseNEW | ElementStateNEW
   response: ResponseFull | null
```

Comment:
Can you add a // TODO: Remove this here  please so we don't forget about it?

Reply:
Do you mean rename to latestApplicationResponse ?

Reply:
I think she just means as TO-DO comment as a marker for things we need to delete later.

Reply:
Sure but I am not sure what to delete later based on the code this comment is pointing to

Reply:
Hmm I think it was just a confusion. I though the new field would be a replacement fro `response` but it isn't , since `thisReviewLatestResponse` is a review instead. 😜